index.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS205: Consider reworking code to avoid use of IIFEs
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
```

Comment:
We can probably delete this block. As an aside, the new `hubot:script` generator probably has a better version of this?

Reply:
Yup did this locally but forgot to do it here.

Reply:
Done :+1: